Extension/src/LanguageServer/configurations.ts

```diff
@@ -998,10 +998,15 @@ export class CppProperties {
                 } else if (configuration.includePath) {
                     // If the user doesn't set browse.path, copy the includePath over. Make sure ${workspaceFolder} is in there though...
```

Comment:
After my current PR on the native side goes in, I want to remove this section, making it the responsibility of the native side to populate the browse paths with the include paths, if the browse path is undefined. That way, I can ensure any include paths ingested via compiler arguments get into the browse path also (if not explicitly set).  Since this would be removed soon (before the next insiders, if possible), maybe this fix isn't needed.

Reply:
@Colengms Sure, yeah, it sounds like this won't be needed.